<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWidgetBacklog</class>
 <widget class="QWidget" name="CWidgetBacklog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelBacklog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>backlog</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="pleBacklog">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>estimated workload(man-day)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leEstimatedWorkload">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:1px solid black;
border-radius:4px;
padding:4px;
}
QPushButton::hover{
border:1px solid black;
border-radius:4px;
background-color:lightgray;
padding:4px
}
QPushButton::pressed{
border:1px solid black;
border-radius:4px;
background-color:gray;
padding:4px
}</string>
       </property>
       <property name="text">
        <string>save</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddTask">
       <property name="styleSheet">
        <string notr="true">QPushButton{
border:1px solid black;
border-radius:4px;
padding:4px;
}
QPushButton::hover{
border:1px solid black;
border-radius:4px;
background-color:lightgray;
padding:4px
}
QPushButton::pressed{
border:1px solid black;
border-radius:4px;
background-color:gray;
padding:4px
}</string>
       </property>
       <property name="text">
        <string>add task</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QFrame" name="frameTasks">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
